pkg/common/malloc/default.go

```diff
@@ -17,9 +17,28 @@ package malloc
 import (
 	"os"
 	"runtime"
+	"strconv"
 	"strings"
+
+	"github.com/matrixorigin/matrixone/pkg/logutil"
+	"go.uber.org/zap"
 )
 
+// checkFraction controls the fraction of checked deallocations
+// On average, 1 / fraction of deallocations will be checked for double free or missing free
+var checkFraction = func() uint32 {
+	// default fraction, large enough to not effecting performance too much
+	fraction := uint32(8192)
+	// control with env var
+	if str := os.Getenv("MO_MALLOC_CHECK_FRACTION"); str != "" {
```

Comment:
Use our config system for this, not os.Getenv

Reply:
done